### DISCUSSION TITLE
Can someone identify why my profile doesn't work?

### DISCUSSION THREAD

**Discussion by AquaCobell** [microsoft/terminal#13913] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/71768923/188223472-a5fcd722-6348-402c-8d99-5122278dc1f5.png)

If i use the command in a blank powershell it works perfectly fine

**Comment by zadjii-msft** [comment]
You haven't given any information on _how_ it doesn't work. What error message does it give you?

**Reply by AquaCobell** [reply]
![image](https://user-images.githubusercontent.com/71768923/188490984-8ff1bad8-7c42-4ee7-a16d-2d5f1c0e0cd2.png)
Aparrently it doesnt recognize the path

**Comment by jameswylde** [comment, marked as answer]
Hi, for the path, please use double "//" and it will resolve correctly:

`C://Users//Nico//Desktop//etc`
